managing_log_files/index.md

```diff
@@ -0,0 +1,6 @@
+---
+---
+
+# Managing Log Files
```

Comment:
NOTE I didn't see any good place to drop in examples of working with log files so thought a new section under "Administration" would work, open to other ideas and/or names for this section

Reply:
Yeah I like troubleshooting, I had been thinking of "Debugging" but that sounded too developer focused for these docs but troubleshooting sounds good.